[2025-06-17 15:54:10,031] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2022-12-17T00:00:00+00:00 [queued]>
[2025-06-17 15:54:10,141] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2022-12-17T00:00:00+00:00 [queued]>
[2025-06-17 15:54:10,141] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:54:10,141] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-17 15:54:10,141] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-17 15:54:10,186] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): thrid_task> on 2022-12-17T00:00:00+00:00
[2025-06-17 15:54:10,206] {standard_task_runner.py:52} INFO - Started process 3835 to run task
[2025-06-17 15:54:10,226] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', '2022-12-17T00:00:00+00:00', '--job-id', '3078', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp4me7d2x8', '--error-file', '/tmp/tmp3j996cjp']
[2025-06-17 15:54:10,237] {standard_task_runner.py:77} INFO - Job 3078: Subtask thrid_task
[2025-06-17 15:54:10,441] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task 2022-12-17T00:00:00+00:00 [running]> on host 4f8b76136637
[2025-06-17 15:54:10,704] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-17T00:00:00+00:00
[2025-06-17 15:54:10,708] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-17 15:54:10,710] {bash.py:158} INFO - Running command: echo hey, I am task3 and will be running after task1 at the same time as task2!
[2025-06-17 15:54:10,732] {bash.py:169} INFO - Output:
[2025-06-17 15:54:10,734] {bash.py:173} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2025-06-17 15:54:10,736] {bash.py:177} INFO - Command exited with return code 0
[2025-06-17 15:54:11,028] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20221217T000000, start_date=20250617T155410, end_date=20250617T155411
[2025-06-17 15:54:11,652] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-17 15:54:11,727] {local_task_job.py:146} INFO - Task exited with return code 0
